From fd814ba54976825875a3c0361f4ef061d411dae5 Mon Sep 17 00:00:00 2001
From: Leander Schulten <Leander.Schulten@rwth-aachen.de>
Date: Tue, 26 Oct 2021 13:44:52 +0200
Subject: [PATCH] vcpkg: add missing include

---
 src/vcpkg.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vcpkg.cpp b/src/vcpkg.cpp
index 7a7eea68c..690ed342f 100644
--- a/src/vcpkg.cpp
+++ b/src/vcpkg.cpp
@@ -22,6 +22,7 @@
 #include <vcpkg/vcpkgpaths.h>
 
 #include <cassert>
+#include <clocale>
 #include <memory>
 #include <random>
 
